docs/spec.yaml

```diff
@@ -10,6 +10,15 @@ repository:
     repo_download:
     repo_download_location:
 
+### Inference
```

Comment:
Why didn't the deepcsr model have this inference field?

Reply:
@hvgazula The deepcsr model does currently have that inference field. However, the spec.yaml template didn't. I just updated the empty template including inference.